pkg/pmem-device-manager/pmme-lvm_test.go

```diff
@@ -0,0 +1,229 @@
+/*
```

Comment:
name of that file seems misspelled, should be pmem-lvm_test.go ?

Reply:
Yes, it should be `pmd-lvm_test.go`. I fixed it now.